docs/releaseNotes.yml

```diff
@@ -84,6 +99,32 @@ items:
           <code>*-migration.yaml</code> files have not been part of the migration instructions, and
           while the <code>*-agent.yaml</code> files remained part of the instructions they were
           actually unnescessary.
+      - title: Metric naming change for grpc_stats
+        type: change
+        body: >-
+          The previous version of $productName$ was based on Envoy 1.17 and when using grpc_stats
+          with <code>all_methods</code> or <code>services</code> set, it would output metrics in
+          the following format <code>envoy_cluster_grpc_{ServiceName}_{statname}</code>. When
+          neither of these fields are set it would be aggregated to <code>envoy_cluster_grpc_{statname}</code>.
+
+          The new behavior since Envoy 1.18 will produce metrics in the following format
+          <code>envoy_cluster_grpc_{MethodName}_statsname</code> and <code>envoy_cluster_grpc_statsname</code>.
+
+          After further investigation we found that Envoy doesn't properly parse service
+          names such as <code>cncf.telepresence.Manager/Status</code>. In the future, we will work
+          upstream Envoy to get this parsing logic fixed to ensure consistent metric naming.
+      - title: Default behavior for empty grpc_stats changed
+        type: change
+        body: >-
+          Previously not setting any values in the <code>grpc_stats</code> field would cause $productName$
+          to throw an error. To ensure that $productName$ does not crash setting the new default will
+          be the equivalent of setting the following:
+
+          <code>
+          config:
+            grpc_stats:
+              all_methods: false
+          </code>
```

Comment:
That won't do multi-line code blocks when rendered to HTML; whitespace'll get munged. Doing multi-line codeblocks in a release notes entry is probably a bad idea anyway--keep it more brief.

Reply:
lmao github rendered `$productName$` as TeX math